netbench/run-netbench-request-response.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+#
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# build the scenario generation tool
+cargo build --manifest-path netbench-scenarios/Cargo.toml
```

Comment:
You should be able to build everything all at once in the workspace

```suggestion
cargo build --release
```